website/versioned_docs/version-0.67/setup-ci.md

```diff
@@ -53,7 +53,7 @@ Now push your changes and the CI pipeline should be up and running.
 ### Storing Certificates Securely
 There are a several options where you can securely store your certificate information:
 
-- [GitHub Secrets](https://docs.github.com/actions/reference/encrypted-secrets)
+- [GitHub Secrets](https://docs.github.com/actions/security-guides/encrypted-secrets)
```

Comment:
these changes seem fine to me but perhaps either a) there is already one place (in GH? or docs.microsoft.com?) we can point people to, or b) we can use shortlinks so we don't have to rev the markdown